docs/source/whatsnew/pr/latex_rendering_relative_files.rst

```diff
@@ -0,0 +1,6 @@
+Relative filenames in Latex rendering
+=====================================
+
+The input and output file arguments to `latex` and `dvipis` are files relative to the current working directory.
```

Comment:
I had a hard time understanding this before I looked through the whole module. I need to nitpick this, sorry.

I don't think this explains the change to the user particularly well. The user doesn't necessarily see `latex` nor `dvipis` commands. For the user, the change is in terms of the API they call.

Could you please explain the change in terms of the API that changes, as opposed to its internals?

Reply:
For the user there are no changes to the API at all, so we could also skip the whatsnew. 

I changed the text to make clear this is an internal change.